res/xml/settings.xml

```diff
@@ -9,6 +9,9 @@
     <juloo.common.IntSlideBarPreference android:key="longpress_timeout" android:title="@string/pref_long_timeout_title" android:summary="%sms" android:defaultValue="600" min="50" max="2000"/>
     <juloo.common.IntSlideBarPreference android:key="longpress_interval" android:title="@string/pref_long_interval_title" android:summary="%sms" android:defaultValue="25" min="5" max="100"/>
     <CheckBoxPreference android:key="precise_repeat" android:title="@string/pref_precise_repeat_title" android:summary="@string/pref_precise_repeat_summary" android:defaultValue="true"/>
+    <CheckBoxPreference android:key="lockable_shift" android:title="@string/pref_lockable_shift_title" android:summary="@string/pref_lockable_shift_summary" android:defaultValue="true"/>
+    <CheckBoxPreference android:key="lockable_ctrl" android:title="@string/pref_lockable_ctrl_title" android:summary="@string/pref_lockable_ctrl_summary" android:defaultValue="true"/>
+    <CheckBoxPreference android:key="lockable_alt" android:title="@string/pref_lockable_alt_title" android:summary="@string/pref_lockable_alt_summary" android:defaultValue="true"/>
```

Comment:
Default value should be the most usual one: true for shift, false for ctrl and alt.